README.md

```diff
@@ -2,14 +2,14 @@
 
 This reference implementation illustrates a basic approach for authoring and running a chat application in a single region with Azure Machine Learning and Azure OpenAI. This reference implementation supports the [Basic Azure OpenAI end-to-end chat reference architecture](https://learn.microsoft.com/azure/architecture/ai-ml/architecture/basic-openai-e2e-chat).
 
-The implementation takes advantage of [Prompt flow](https://microsoft.github.io/promptflow/) in [Azure Machine Learning](https://azure.microsoft.com/products/machine-learning) to build and deploy flows that can link the following actions required by an LLM chat application:
+The implementation takes advantage of [Prompt flow](https://microsoft.github.io/promptflow/) in [Azure Machine Learning](https://azure.microsoft.com/products/machine-learning) to build and deploy flows that can link the following actions required by a generative AI chat application:
 
 - Creating prompts
 - Querying data stores for grounding data
 - Python code
-- Calling Large Language Models (LLMs)
+- Calling language models (such as ChatGPT)
```

Comment:
```suggestion
- Calling language models (such as GPT-4)
```

Reply:
We aren't using 4. We're using 3.5-Turbo.  And I left it generic, so it wasn't a maintenance chase.

Do you want to see something changed here?

Reply:
Yes, chatGPT is an application, not a model. That is why I suggested a model. I'd update it to Turbo.

Reply:
Updated to GPT models